src/tests/ftest/launch.py

```diff
@@ -287,7 +287,8 @@ def set_provider_environment(interface, args):
                 for line in output_data[0][0]:
                     provider = line.decode("utf-8").replace(":", "")
                     # Temporary code to only enable verbs on HW Large stages
-                    if "verbs" in provider and "hw" in tags and "large" in tags:
+                    if "verbs" in provider and "hw" in tags and (
+                            "large" in tags or "small" in tags):
```

Comment:
This change was recently landed in commit 2b0ddd065a7c0c39bfc06f249dc654e998b9c915